README.md

```diff
@@ -663,6 +663,34 @@ groupsRes.data.forEach((group) => {
 });
 ```
 
+### Manage Flows
+
+You can import and export flows and screens, or the project theme:
+
+```typescript
+// Export the flow and it's matching screens based on the given id
+const res = await descopeClient.management.flow.export('sign-up');
+console.log('found flow', res.flow);
+res.screens.forEach((screen) => {
+  // do something
+});
+
+// Import the given flow and screens as the given id
+const updatedRes = descopeClient.management.flow.import('sign-up', flow, screens);
```

Comment:
not critical, but I would add above this line

```
const flow = res.data.flows;
const screens = res.data.screens;
```

or something similar, cuz it is not expected from dev to build it manually